packages/dmn-editor/tests/e2e/__fixtures__/palette.ts

```diff
@@ -32,6 +32,7 @@ export class Palette {
           .dragTo(this.diagram.get(), { targetPosition: args.targetPosition });
 
         if (args.thenRenameTo) {
+          await this.nodes.waitForNodeToBeFocused({ name: DefaultNodeName.INPUT_DATA });
```

Comment:
However, for a scenario, where one test add two input data nodes, I needed to add this wait, I think we still have some 'delay' for input nodes, once we add them to canvas? Probably not observable by human, only by automatization?